backend/database/migrations/2017_11_24_140740_create_initial_schema.php

```diff
@@ -72,11 +72,11 @@ public function up()
 
         Schema::create('design_asset', function (Blueprint $table) {
             $table->increments('id');
-            $table->integer('brief_id')->unsigned();
+            $table->integer('design_id')->unsigned();
```

Comment:
Updating migrations will not work for people who have already run them. Can you create a new migration that changes the fields?